packages/spotlight/SpotlightContainerDecorator/tests/SpotlightContainerDecorator-specs.js

```diff
@@ -24,186 +27,154 @@ describe('SpotlightContainerDecorator', () => {
 
 	test('should set itself as the active container on mouse enter', () => {
 		const Component = SpotlightContainerDecorator(Div);
+		render(<Component data-testid={testId} spotlightId="test-container" />);
+		const component = screen.getByTestId(testId);
```

Comment:
Let's add some space around render function.

```suggestion
		
		render(<Component data-testid={testId} spotlightId="test-container" />);

		const component = screen.getByTestId(testId);
```